packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -121,34 +121,31 @@ function buildInvitationsFromInput({
   });
 }
 
-export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) => {
-  const myLog = log.getSubLogger({ prefix: ["inviteMemberHandler"] });
-  const translation = await getTranslation(input.language ?? "en", "common");
-  await checkRateLimitAndThrowError({
-    identifier: `invitedBy:${ctx.user.id}`,
-  });
-
-  const invitations = buildInvitationsFromInput({
-    usernameOrEmail: input.usernameOrEmail,
-    roleForAllInvitees: input.role,
-  });
-
-  const team = await getTeamOrThrow(input.teamId);
+type TargetTeam =
+  | {
+      teamId: number;
+    }
+  | {
+      team: TeamWithParent;
+    };
```

Comment:
Support passing team directly if it is available or just teamId so that it can fetch it itself